src/test/java/org/simlar/simlarserver/services/pushnotification/ApplePushNotificationMockServerTest.java

```diff
@@ -35,47 +38,84 @@ public void setup() {
         mockServer = startClientAndServer();
     }
 
-    @Test
-    public void testRequestAppleVoipPushNotification() {
+    private void createMockServerRequest(final String deviceToken, final String expiration, final HttpResponse response) {
```

Comment:
Would be cool if the signatures of `createMockServerRequest` and `requestVoipPushNotification` did actually match regarding the `expiration` parameter type.